blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,169 @@
+    responder.reply(replies)
+
+@app.handle(intent='routing_number')
+def faq_routing_number_handler(request, responder):
+    replies = ['Your routing number is 121122676.']
+    responder.reply(replies)
+
+@app.handle(intent='fraud_charge')
+def faq_fraud_charge_handler(request, responder):
+    replies = ['If you think someone used your account without your permission or notice any charge errors, tell us immediately by calling 1-800-MindMeld.']
+    responder.reply(replies)
+
+@app.handle(intent='forgot_pin')
+def faq_forgot_pin_handler(request, responder):
+    replies = ['If you\'ve forgotten your PIN or don\'t have one, please visit your local branch or contact customer service at 1-800-MindMeld to have a new PIN mailed to you.']
+    responder.reply(replies)
+
+@app.handle(intent='apply_loan')
+def faq_apply_loan_handler(request, responder):
+    replies = ['To apply for a loan or any further questions you will need to visit our website MindMeld.com/loans']
+    responder.reply(replies)  
+
+@app.handle(intent='activate_creditcard')
+def faq_activate_creditcard_handler(request, responder):
+    replies = ['You can activate your card by logging into our website and also over the phone. Just call the MindMeld card activation number 1-800-MindMeld. Follow the prompts that will trigger the activation of your card.']
+    responder.reply(replies)
+
+@app.handle(default=True)
+def default(request, responder):
+    replies = ['Sorry, I didn’t get that.', "I'm afraid I don't understand.",
+               'Sorry, say that again?', 'Sorry, can you say that again?',
+               "I didn't get that. Can you say it again?",
+               'Sorry, could you say that again?', 'Sorry, can you tell me again?',
+               'Sorry, tell me one more time?', 'Sorry, can you say that again?']
+    responder.reply(replies)
+
+
+
+entity_form = [
+
+    FormEntity(
+        entity='account_type',
+        role='origin',
+        responses=['Sure. Transfer from which account?']),
+    FormEntity(
+        entity='account_type',
+        role='dest',
+        responses=['To which account?']),
+    FormEntity(
+        entity='sys_amount-of-money',
+        responses=['And, how much do you want to transfer?'])
+    ]
+                       
+
+@app.auto_fill(intent='transfer_balances', entity_form=entity_form)
+def transfer_balances_handler(request, responder):
+	
+    for entity in request.entities:
+        if entity['type'] == 'account_type':
+            if entity['role'] == 'origin':
+                responder.slots['origin'] = entity['value'][0]['cname'] or entity['text']
+            elif entity['role'] == 'dest':
+                responder.slots['dest'] = entity['value'][0]['cname'] or entity['text']
+        else:
+            print(entity)
+            responder.slots['amount'] = entity['text']
+
+    replies = ["All right. So, you're transferring {amount} from your "
+               "{origin} to a {dest}. Is that right?"]
```

Comment:
Unless you have another intent to handle yes or no responses, or any follow-up intents, you can terminate this dialogue here.